content/05.methods.md

```diff
@@ -71,20 +71,23 @@ We assess both the ability to predict the presence of an SV as well as the full
 For the *presence* evaluation, both heterozygous and homozygous alternate SVs are compared jointly using the approach described above.
 To compute genotype-level metrics, the heterozygous and homozygous SVs are compared separately.
 Before splitting the variants by genotype, consecutive heterozygous variants are first merged if located at less that 20 bp from each other.
+[//]: # (I am still a bit confused about this. Is it only the truth set variants where the phase might be known that are merged? If it is also the variants in the call set I would be a bit worried since they might not be phased, for example BayesTyper will alwayes give 0/1 and never 1/0. If call set variants are also merged, I think it is something that will need to be changed at some point.)
```

Comment:
Both the truth set and call set are transformed like that and the phase information is not used here. The idea is to catch calls that might be fragmented. It's assuming that two het variants at less than 20 bp from each other are likely the same variant. Sometimes it's not and it's incorrect to stitch them but sometimes it should be because it's the same variant fragmented into pieces in the catalog or in the graph. If they shouldn't be stitch (and the genotype predictions are correct), they will be stitched in both truthset and callset, and we would "just" count 1 TP instead of 2 TPs. If we rightly stitch them, we'll potentially rescue wrongly assigned FP/FN. 

Maybe using a more stringent distance, e.g. 5 bp from each other, would be less disturbing. Anyway that was the rationale but we can remove this step if it's feels wrong. In practice the "het merging" is more important and has a bigger effect on the evaluation results:

![sveval-stitch-merge](https://user-images.githubusercontent.com/5704457/57964475-fdf6c880-78ea-11e9-887e-2a63cd42c662.png)

On a side note vg, Delly and SVTyper (maybe SMRT-SV too?) also use `0/1` for het so I don't think BayesTyper is penalized. And it's not using the phase information anyway, it's really stitching hets, not stitching haplotypes. 

Hope this clarifies, let me know if you think I should rerun the evaluation without this step (it's easy).

Reply:
Thanks for the clarification. The reasoning behind it makes more sense now. I still think it would be better without it, but it is also completely fine with me if you decide to keep it. However, it might then be good idea to write a bit more in the methods section on why you do it.